examples/4_Sustainability/supporting-files/bom-2301-assembly.xml

```diff
@@ -120,7 +120,7 @@
 										<dbKey xmlns="http://www.grantadesign.com/12/05/GrantaBaseTypes">MI_Restricted_Substances</dbKey>
 										<recordGUID xmlns="http://www.grantadesign.com/12/05/GrantaBaseTypes">92f90382-610a-4737-a18f-b153ea88e4ad</recordGUID>
 									</MIProcessReference>
-									<Identity>Primary processing, Metal extrusion, hot</Identity>
+									<Identity>Metal extrusion, hot</Identity>
```

Comment:
This change isn't technically necessary. It was believed that the identity would override the name in the summary output, but the name in the database is always used instead.